test/distributed/test_distributed_spawn.py

```diff
@@ -2,12 +2,15 @@
 
 import os
 import sys
+import subprocess
+
 
 import torch
 import torch.distributed as dist
 
-os.environ["BACKEND"] = "xccl"
-os.environ["WORLD_SIZE"] = str(torch.accelerator.device_count())
+# os.environ["BACKEND"] = "xccl"
```

Comment:
Will the "BACKEND" and "WORLD_SIZE" be set automatically in current implementation? I see there is a checker in the test for the environments.